### PR TITLE
supports running ApiTest in different cultures

### PR DESCRIPTION
**Pull request by davidalpert**
Running this code from Canada (en-CA), import of the included 'sample.qif' failed with an `InvalidCastException`

### CODE REVIEW

**Comment by davidalpert** [comment]
I will check that failing test tomorrow; feedback welcome.

**Comment by davidalpert** [comment]
Hmm... open to suggestions.

**Comment by shanewalters** [comment]
Thank you for the PR, this looks great! For the failing test `Cannot_import_sample_qif_when_current_culture_is_en_CA`, I'm a little confused. I think the PR is designed to solve the problem the unit test is expecting as a problem, and thus fails. If that's true, please remove the test or modify it to expect a pass condition. Once that's fixed, I'll give it another review. Thanks!

**Comment by shanewalters** [comment]
Also, please add the NuGet package `NUnitTestAdapter.WithFramework` to `QifApi.Tests` as part of the PR. Thanks!

File: QifApi.Tests/ImportTests.cs [hunk 1]
```diff
@@ -32,6 +33,7 @@ public void Cannot_import_sample_qif_when_current_culture_is_en_CA()
             {
                 Assert.Throws<InvalidCastException>(() =>
```
Comment (davidalpert) [inline QifApi.Tests/ImportTests.cs]: This assertion passes on my machine.  It is intended to explicitly set the current thread's culture to en-CA, which is what I assume my local machine to be using, hence why I discovered this issue in the first place.  I can remove it but will think more about what else could cause this to pass on my machine and fail on the build server.  Does it run on your machine?

**Comment by davidalpert** [comment]
those two tests changed in that last commit pass now on the server yet fail on my machine. that tells me that I have not yet found nor solved the root of the problem.

**Comment by shanewalters** [comment]
I'll go ahead and accept the PR. If you find what causes the tests to fail on your machine, feel free to post additional updates.

**Pull request merged**